src/components/DisplayBook/index.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import ProgressBar from 'react-customizable-progressbar';
+import ReactLoading from 'react-loading';
+import removeBook from '../../redux/books/books'
+
+const DisplayBook = ({
+  book: {
+    id, title, category, author, completed, currentChapter,
+  },
+}) => {
+  const dispatch = useDispatch();
+  const [requestStatus, setRequestStatus] = useState('idle');
+
+  const handleDelete = (id) => {
+    setRequestStatus('loading');
+    dispatch(removeBook(id));
+
+    setTimeout(() => {
+      setRequestStatus('idle');
+      const { location } = window;
+      location.reload();
+    }, 4000);
+  };
+
+  const renderLoading = () => {
+    const spinnerType = 'spinningBubbles';
+    const spinnerColor = '#0290ff';
+    return (
+      <div style={{ display: 'flex', justifyContent: 'center', alignItems: 'center' }}>
+        <ReactLoading
+          type={spinnerType}
+          color={spinnerColor}
+          height="10%"
+          width="10%"
+        />
+      </div>
+    );
+  };
+
+  const renderContainer = () => (
+    <div className="container group">
+      <div className="left">
+        <p className="category">{category}</p>
+        <p className="title">{title}</p>
+        <p className="author">{author}</p>
+        <br />
+        <div className="bottom-buttons">
+          <button type="button" className="buttons">Comments</button>
+          <span>&#124;</span>
+          <button type="submit" className="buttons" onClick={() => handleDelete(id)}>Remove</button>
+          <span>&#124;</span>
```

Comment:
- [ ] The functionality on the handleDelete function passed here does not seem to be removing books. It removes a book from the screen and constantly shows the spinner like forever and when I refresh the browser, the deleted book still shows on the screen again. Kindly work on that functionality to work as expected 👍